micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSView.js

```
@@ -125,30 +117,93 @@
+  }
+
+  const locale = "en_IN";
+  const localizationReqCriteria = {
+    url: `/localization/messages/v1/_search?locale=${locale}&tenantId=${tenantId}&module=${localizationModule}`,
+    params: {},
+    body: {
+    },
+    config: {
+      enabled: !!data && !!MdmsRes && !!data?.schemaCode && !!tenantId && localizationCodes.length > 0,
+      select: (respData) => {
+        let messageMap = {};
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'messageMap' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)